MpsatSynthesisPlugin/src/org/workcraft/plugins/mpsat/tasks/MpsatSynthesisChainResult.java

```diff
@@ -25,12 +24,11 @@ public MpsatSynthesisChainResult(Result<? extends Object> exportResult,
     }
 
     public MpsatSynthesisChainResult(Result<? extends Object> exportResult,
-            Result<? extends ExternalProcessResult> pcompResult,
             Result<? extends ExternalProcessResult> punfResult,
             Result<? extends ExternalProcessResult> mpsatResult,
             MpsatSynthesisSettings mpsatSettings) {
 
-        this(exportResult, pcompResult, punfResult, mpsatResult, mpsatSettings, null);
```

Comment:
As with the other parameter that you say is needed, it actually isn't in practice. The receiver never uses it.